bin/skyfactory/select_and_merge_samples.py

```diff
@@ -198,7 +198,7 @@ def pair_files(ofiles, tfiles, pzfiles):
 
         obs = obsf[-1].read()
         truth = truthf[-1].read(columns=['ID', 'GAMMA1',
-                                         'GAMMA2', 'KAPPA', 'Z'])
+                                         'GAMMA2', 'KAPPA', 'Z', 'Z_COS'])
```

Comment:
If not adding z_cos, there will be error showing the code cannot find Z_COS in truth.